cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Connectors.java

```diff
@@ -114,6 +114,11 @@ public static KafkaMirrorMaker2Connectors fromCrd(Reconciliation reconciliation,
                     } else if (!existingClusterAliases.contains(mirror.getTargetCluster())) {
                         errorMessages.add("Target cluster alias " + mirror.getTargetCluster() + " is used in a mirror definition, but cluster with this alias does not exist in cluster definitions");
                     }
+
+                    String connectCluster = kafkaMirrorMaker2.getSpec().getConnectCluster();
```

Comment:
If there will be multiple sets of mirrors, you will be extracting this again and again. Would it make sense to get it only once before we start the for loop?